addons/website/models/ir_ui_view.py

```diff
@@ -156,7 +156,7 @@ def unlink(self):
                     view.with_context(website_id=website.id).write({'name': view.name})
 
         specific_views = self.env['ir.ui.view']
-        if self and self.pool._init and not self._context.get('from_cow'):
```

Comment:
@rdeodoo Didn't we find a reason to not do that ? x) We talked about it... but I can't remember now.